### ISSUE TITLE
Cannot purchase mount training (Elysium: Boonotcool)

### ISSUE THREAD

**Issue by dtabacaru** [elysium-project/server#2550]
Character "Boonotcool" (lvl 40 gnome mage) cannot purchase mount training (NPC does not show it available). NPC seems to think I may not be a gnome (claims I am too large). The WoW API function GetRace() does seem to correctly return "Gnome".

![image](https://user-images.githubusercontent.com/21049690/30345001-8435b278-97d1-11e7-9128-a9fb947a9e51.png)

![image](https://user-images.githubusercontent.com/21049690/30345005-8af6ce44-97d1-11e7-8fd9-668b857ea54c.png)

Character has other small quirks too - wizard oil cannot be applied to weapon, lock pick does not work through trade.

The most important to me is, of course, the mount though...

Let me know if there's anything else you need. I could potentially look into a fix myself provided someone points me in the right direction : - ) (probably not going to dig through the code myself though).

**Comment by Phoosy** [comment]
Wizard oil is disabled on Elysium - the recipe was made available too early so a hackfix was to just disable application.

Have you logged off and deleted your WDB folder in its entirety?

**Comment by dtabacaru** [comment]
Yes, I've tried twice now on two different machines with WDB deleted before logging in.

Same result.

**Comment by Brautito** [comment]
Hey I had the same problem and I fixed it with a re-install.  Maybe give that a try if you are still having this issue.
